reference-implementation/__tests__/resolving-builtins.js

```diff
@@ -155,4 +168,16 @@ describe('Fallbacks with built-in module addresses', () => {
   it(`should fall back past "${NONE}"`, () => {
     expect(resolveUnderTest('none')).toMatchURL('https://example.com/app/none-fallback.mjs');
   });
+
+  it('should fall back for [built-in, built-in]', () => {
+    expect(resolveUnderTest('twoGoodBuiltins')).toMatchURL(BLANK);
+  });
+
+  it('should fail for [built-in, fetch scheme, fetch scheme]', () => {
```

Comment:
These aren't quite correct. resolve() would return the first URL which was successfully fetched, most likely. The complexities there (e.g. making resolve() async or network dependent) are why we've kept this as unimplemented. It'd be better to stick to only supporting the [built-in, fetch scheme URL] pattern for now in the fallbacks.

Reply:
For simplicity, this implementation assumes that all web resource fetches succeed (which seems to match the current tested behavior). We can artificially restrict the resolution algorithm to only accept fallback lists supported by the current implementation. Or we could use some different strategy to determine a success status.